fix: Colour ignored when `0` is used as british-colour in Embed (#3256)
Pycord-Development/pycord: CHANGELOG.md

  ([#3231](https://github.com/Pycord-Development/pycord/pull/3231))
- Allow `ForumTag` to be created without an emoji.
  ([#3245](https://github.com/Pycord-Development/pycord/pull/3245))

> - Fix issue where `0` would be ignored as an `Embed`'s `colour` parameter.

Reviewer: ```suggestion
- Fix an issue where an `Embed` object's `colour` parameter would be ignored when set to `0`.
```
